contents/articles/2016-11_focal-musical-expression/index.md

```diff
@@ -0,0 +1,37 @@
+--- 
+title: An Eye-Tracking Musical Expression Controller
+date: Fri Nov 11 2016 14:43:37 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Focal is an experimental concept that uses AR headset with eye tracking as a digital musical controller..."
+featured_img: "../../../../media/images/blog/focal-systemb.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/focal-music.jpg"
+---
+
+Focal is an exciting experimental concept that uses AR headset with eye tracking as a digital musical controller. Enabling the musician to control many virtual electronic audio effects and synthesis parameters. The performer can control the virtual dials and switches by glancing at the effects they want to activate and using a single expression pedal to adjust those effects, while maintaining posture and balance for minimal disturbance and interaction during performance.
```

Comment:
Provide a link to their paper: http://stewartgreenhill.com/documents/FocalEyeTrackingMusicalExpressionController-NIME2016.pdf and cite the two authors.